test/test_torch.py

```diff
@@ -2890,6 +2891,35 @@ def add_neg_dim_tests():
         assert not hasattr(AbstractTestCases._TestTorchMixin, test_name), "Duplicated test name: " + test_name
         setattr(AbstractTestCases._TestTorchMixin, test_name, make_neg_dim_test(name, tensor_arg, arg_constr, types, extra_dim))
 
+# Tests Vital Signs for Torch
+class TestVitalSigns(TestCase):
+    def test_basic_vitals(self):
+        # Protect current environment variable, if set
+        stash = None
```

Comment:
given that you'll likely expand the tests, it makes sense to write a context manager that would handle resetting current TORCH_VITAL env, see e.g. `tf32_off` context manager

Reply:
Thanks for the tip! This really cleaned up the code.